### PR TITLE
Remove defunct GOC resources

### PR DESCRIPTION
**Pull request by brianhlin**
Notably the following CE resources (@djw8605 is this ok with GRACC?):

- IUB-VTB
- IUB_ITB
- IUB_ITB2
- IUB_ITB_AG
- IUPUI-ITB
- IU_COBRA_ITB
- IU_OSG
- IU_iuatlas
- GOC_ITB_CE
- IPAS_OSG
- Soichi_Test_Resource

I also removed Scott as submitter contact for a few resources and a few IU GOC staff from the MIS OASIS VO manager list. Scott is still an OASIS manager for nanoHub and ILC but I'm not sure whether or not he still needs to be.

### CODE REVIEW

File: topology/Indiana University/IU UITS/Condor collectors.yaml [hunk 1]
```diff
@@ -1,52 +0,0 @@
-Disable: false
```
Comment (matyasselmeci) [inline topology/Indiana University/IU UITS/Condor collectors.yaml]: This should be moved to where the collectors actually are (Nebraska?) and contact & support center info should be updated.

Comment (djw8605) [reply topology/Indiana University/IU UITS/Condor collectors.yaml]: yes, nebraska.

File: topology/Indiana University/IU UITS/GOC_OIM.yaml [hunk 2]
```diff
@@ -1,39 +0,0 @@
-Disable: false
```
Comment (matyasselmeci) [inline topology/Indiana University/IU UITS/GOC_OIM.yaml]: This should also be moved to Nebraska and updated.

File: topology/Indiana University/IU UITS/GOC_Repo.yaml [hunk 3]
```diff
@@ -1,68 +0,0 @@
-Disable: false
```
Comment (matyasselmeci) [inline topology/Indiana University/IU UITS/GOC_Repo.yaml]: Move & update.

File: topology/Indiana University/IU UITS/GOC_Repo_ITB.yaml [hunk 4]
```diff
@@ -1,29 +0,0 @@
-Disable: false
```
Comment (matyasselmeci) [inline topology/Indiana University/IU UITS/GOC_Repo_ITB.yaml]: Move & update

File: topology/Indiana University/IU UITS/MyOSG.yaml [hunk 5]
```diff
@@ -1,60 +0,0 @@
-Disable: false
```
Comment (matyasselmeci) [inline topology/Indiana University/IU UITS/MyOSG.yaml]: Move & update

File: topology/Indiana University/IU UITS/OSG Oasis.yaml [hunk 6]
```diff
@@ -1,39 +0,0 @@
-Disable: false
```
Comment (matyasselmeci) [inline topology/Indiana University/IU UITS/OSG Oasis.yaml]: Move & update

File: topology/Indiana University/IU UITS/Condor collectors.yaml (hunk 1 above)
Comment (brianhlin) [reply topology/Indiana University/IU UITS/Condor collectors.yaml]: Here? https://github.com/opensciencegrid/topology/tree/master/topology/University%20of%20Nebraska/Nebraska-Lincoln

Comment (djw8605) [reply topology/Indiana University/IU UITS/Condor collectors.yaml]: I think it should be the Omaha entry:
https://github.com/opensciencegrid/topology/tree/master/topology/University%20of%20Nebraska/Nebraska-Omaha

**Comment by brianhlin** [comment]
Don't forget all the perfsonar nodes, OSG display, DOCDB, and message bus. But I'm not sure getting the new resources up in the topology data needs to be in this PR. It doesn't seem like people are using this data, anyway.

**Comment by matyasselmeci** [comment]
Let's get to #72 pretty quickly then.

**Comment by brianhlin** [comment]
Though before we merge master and then merge back into master, I'd like @djw8605 to weigh in on all the CE resources that we'd be removing.

**Pull request merged**